File: local-data-persistent/LatihanRoom/java/app/src/main/java/com/dicoding/picodiploma/mynoteapps/helper/DateHelper.java
package com.dicoding.picodiploma.mynoteapps.helper;

import java.text.DateFormat;
import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.Locale;

public class DateHelper {

    public static String getCurrentDate() {
        DateFormat dateFormat = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss", Locale.getDefault());
        Date date = new Date();

        return dateFormat.format(date);
    }
}

File: local-data-persistent/LatihanRoom/kotlin/app/src/main/java/com/dicoding/picodiploma/mynoteapps/helper/DateHelper.kt
package com.dicoding.picodiploma.mynoteapps.helper

import java.text.SimpleDateFormat
import java.util.*

object DateHelper {
    fun getCurrentDate(): String {
        val dateFormat = SimpleDateFormat("yyyy/MM/dd HH:mm:ss", Locale.getDefault())
        val date = Date()
        return dateFormat.format(date)
    }
}